wiki/Miscellaneous_official_projects/upppy/en.md

```diff
@@ -0,0 +1,42 @@
+---
+tags:
+  - upppy
+  - up.ppy
+  - up.ppy.sh
+---
+
+# upppy
+
+::: Infobox
+![Snapshot of the up.ppy.sh page](img/upppyScreenshot.png "Snapshot of the upppy website, as taken in 2023 by the Internet Archive")
+:::
+
+**up.ppy.sh**, often referred to as **upppy**, was a file uploading service ran and hosted by ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2). It was first established in January 2007, predating osu! itself[^upppy-launch-date-ref], and was mainly used to share images and files within the osu! community.
+
+## Service
+
+upppy was created as a free, lightweight alternative to proprietary image hosting services such as Rapidshare or ImageShack.[^upppy-image-hosting-ref]
+
+The service didn't require any registration to use and allowed for everyone to upload any file (within its Terms of Service) anonymously. Similar to certain [imageboard sites](https://en.wikipedia.org/wiki/Imageboard), upon uploading a file, users were given a choice as to whether to post their file in the default, catch-all working channel (`#random`) or in the one of the more specific channels as follows:
+
+   - `#anime`
+   - `#game`
+   - `#highres`
+   - `#music`
+   - `#photography`
+   - `#screenshots`
+   - `#torrents`
+   - `#tv`
+   - `#video`
+   - `#wallpaper`
+
+Once a file was posted, it became a permanent part of the platform and couldn't be deleted or edited by the uploader. However, much like on [Reddit](/wiki/Community/Reddit), these posts could still receive comments and be upvoted/downvoted by other users.
+
+## Decommissioning
+
+As upppy relied upon [Adobe Flash Player](https://en.wikipedia.org/wiki/Adobe_Flash_Player) to upload user files, the upppy website was effectively rendered read-only starting from December 31, 2020 when [Adobe Flash Player was officially shut down](https://www.adobe.com/id_en/products/flashplayer/end-of-life-alternative.html). Sometime in early 2025, the entire website was taken down completely, leaving only an error page when accessed.
```

Comment:
```suggestion
As upppy used [Adobe Flash Player](https://en.wikipedia.org/wiki/Adobe_Flash_Player) in the file upload form, it was effectively rendered read-only starting from December 31, 2020 when [Adobe Flash Player was officially shut down](https://www.adobe.com/id_en/products/flashplayer/end-of-life-alternative.html). Sometime in early 2025, the entire website was taken down completely, leaving only an error page when accessed.
```

Reply:
> it was effectively rendered read-only starting from December 31, 2020

I don't think it is true -- the screenshot directly contradicts this, showing an upload from 2022. maybe say that it took a significant popularity hit or something instead?